### PR TITLE
CF-375 CF instance migration does not consider existing images

### PR DESCRIPTION
**Pull request by roman-verchikov**
Instance migration would fail if image in destination is deleted
using glance API, but exists in DB.

### CODE REVIEW

File: cloudferrylib/os/actions/transport_instance.py [hunk 1]
```diff
@@ -17,9 +17,13 @@
 
 from cloudferrylib.base.action import action
 from cloudferrylib.os.identity import keystone
+from cloudferrylib.utils import log
```
Comment (SVilgelm) [inline cloudferrylib/os/actions/transport_instance.py]: i suggest
```python
import logging

LOG = logging.getLogger(__name__)
```

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
:+1: wait testing

**Comment by yosharma** [comment]
+1, Tested on staging environment. Its working fine.

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**